amplipi/updater/static/index.html

```diff
@@ -84,6 +87,20 @@ <h1><span class="text-white">Ampli</span><span class="text-danger">Pi</span></h1
               </div>
               <a id="password" role="button" class="btn btn-lg btn-primary text-center mt-3" onClick="set_password();">Update Password</a>
             </div>
+            <div class="tab-pane fade text-left" id="support-tunnel" role="tabpanel" aria-labelledby="support-tunnel-tab">
+              <div id="support-tunnel-dialog">
+                This dialog allows you to request a support tunnel. Once requested, you must give the tunnel ID and preshared key to support, who can be reached via <a href="mailto:support@micro-nova.com">support@micro-nova.com</a>.
```

Comment:
Any chance we can embed the key in this uri?

href="mailto:support@micro-nova.com?subject=Support%20Tunnel%20Request&body=KEY"

Reply:
If thats a bigger lift we can save it for later.